content/en/apps/tutorials/sms-schedules.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Building SMS Forms"
+linkTitle: SMS Schedules
+weight: 6
+description: >
+  Building CHT application SMS forms
+relatedContent: >
+  apps/reference/app-settings/schedules
+
+---
+
+SMS schedules allow you to send reminder messages at predetermined times. SMS schedules can be triggered by [SMS forms]({{< ref "apps/tutorials/sms-forms" >}}) or App forms.
+
+## Purpose of the Tutorial
+
+This tutorial will take you through how to set up SMS schedules for CHT applications.
+
+You will be setting up pregnancy follow up reminders for a pregnancy registration workflow that allows Community Health Workers to register new pregnancies for the household members and get reminders to follow up on the pregnancy.
```

Comment:
I pulled this sentence into the previous line, so it can be deleted if you like the change

Reply:
Done. Yeah, the updated one is more succinct.